### PR TITLE
Brach and jess

### PR DESCRIPTION
**Pull request by sirbrachthepale**
mw backend shit fuck balls

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +32,7 @@
     "pg": "^8.8.0",
     "@babel/preset-env": "^7.20.2",
     "@babel/preset-react": "^7.18.6",
+    "@types/pg": "^8.6.6",
```
Comment (dnvt) [inline package.json:35]: We don't need this

File: package.json [hunk 2]
```diff
@@ -21,6 +21,7 @@
   "homepage": "https://github.com/Skechphora/Skechphora#readme",
   "dependencies": {
     "@reduxjs/toolkit": "^1.9.1",
+    "pg": "^8.8.0",
```
Comment (dnvt) [inline package.json:24]: Remove this :)

File: server/server.js [hunk 3]
```diff
@@ -1,6 +1,14 @@
 const express = require('express');
 const path = require('path');
 
+// const app = express();
```
Comment (dnvt) [inline server/server.js:4]: Can we remove?

File: server/server.js [hunk 4]
```diff
@@ -14,9 +22,30 @@ app.use(express.urlencoded({ extended: true }));
 
 app.use('/transfer', transformDataRouter);
 
+
+
+//at homepage - on endpoint main, send main page to front end
```
Comment (dnvt) [inline server/server.js:27]: Remove this

**Review by dnvt — CHANGES_REQUESTED** [review]
Server.js and package has to be updated

**Comment by jesswang-dev** [comment]
Hi Franswa, we resolved the comments you made.

**Review by dnvt — APPROVED** [review]
LGTM

**Pull request merged**